packages/xen-api/src/index.js

```diff
@@ -99,6 +99,10 @@ export class Xapi extends EventEmitter {
     this._sessionId = undefined
     this._status = DISCONNECTED
 
+    this._lastCatchedEventError = null
+    this._lastFailedFetchTime = undefined
```

Comment:
Is it used?

If not, remove it, we'll add it if/when we need it :slightly_smiling_face: